packages/lit-dev-content/site/tutorials/content/working-with-lists/01.md

```diff
@@ -0,0 +1,51 @@
+When not working with an array as a source of data, an array of templates to
+render can be manually built, for example, by using a `for ... of` loop.
```

Comment:
I didn't notice the example had changed from an array to a `Set` at first, so might be worth calling attention to that change first?
```suggestion
Sometimes you may want to render a list from something other than
an array, such as a `Set`. Note we've changed the example
data from an array to a `Set`.

You can manually manually build an array of templates from the `Set`
by using a `for ... of` loop, for example.
```

Reply:
+1

Reply:
Fixed!